controllers/tinyurl.js

```diff
@@ -1,22 +0,0 @@
-let UrlData = require('../models/db');
-
-module.exports = {
-    index: {
-        get(req, res) {
-        //    var mapurl;
-            var query = UrlData.findOne({ 'shorturl': req.params.url },
-                function(err, doc) {
-                    if (err) {
-                        res.send(err)
-                    }
-                    console.log(req.params);
-                    console.log(doc);
```

Comment:
even if you have error this line will execute best practice is to return after every error in scope.